packages/request-handler/src/notifications.ts

```diff
@@ -0,0 +1,20 @@
+import dayjs from "dayjs";
+
+import type { IntegrationKindByCategory } from "@homarr/definitions";
+import type { Notification } from "@homarr/integrations";
+import { createIntegrationAsync } from "@homarr/integrations";
+
+import { createCachedIntegrationRequestHandler } from "./lib/cached-integration-request-handler";
+
+export const notificationsRequestHandler = createCachedIntegrationRequestHandler<
+  Notification[],
+  IntegrationKindByCategory<"notifications">,
+  Record<string, never>
+>({
+  async requestAsync(integration) {
+    const integrationInstance = await createIntegrationAsync(integration);
+    return await integrationInstance.getNotificationsAsync();
+  },
+  cacheDuration: dayjs.duration(30, "seconds"),
```

Comment:
It's this a bit too aggressive? In my opinion, 5 minutes should be plenty!
Remember, Homarr will fetch even if there is no user "active", meaning that we need to be careful with setting too low intervals.